src/exercises/exerciseTypes/BottomInput.js

```diff
@@ -39,12 +40,11 @@ export default function BottomInput({
     : bookmarksToStudy[0].from_lang;
 
   function handleHint() {
-    setUsedHint(true);
-
-    if (exerciseType === EXERCISE_TYPES.translateWhatYouHear) {
+    if (exerciseType === EXERCISE_TYPES.translateWhatYouHear && firstHint) {
+      setFirstHint(false);
       onHintUsed();
-      setMessageToAPI(messageToAPI + "H");
```

Comment:
This way we never know if they use multiple-hints. I think it's good to have an idea if users go all the way. Do they try to answer after getting the first hint? It would be a different pattern we can't monitor this way.